packages/protocol/contracts/dapis/RrpDapiServer.sol

```diff
@@ -28,13 +30,41 @@ contract RrpDapiServer is MeanMedianHybrid {
 
   AirnodeRrp public airnodeRrp;
   mapping(bytes16 => Dapi) private dapis;
+  // TODO: are we missing a dapiIds bytes16 array for getting all the dapi ids?
```

Comment:
Is there a situation where we would need to get all `dapiIds`? Or is it just good practice to do so?

Reply:
Ahh, I think there isn't a good way to "take a look" and see which dAPIs we've registered, right? Unless we have an array?